recipes/objectlist/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
The license file is already in the source distribution, so you shouldn't include a copy of it here. Please remove this from the PR.

Reply:
Thanks! I removed the license file.